src/sentry.ts

```diff
@@ -64,10 +64,13 @@ export const init = (options: SentryExpoNativeOptions = {}) => {
   }
 
   // Check if build-time update
-  nativeOptions.dist =
-    nativeOptions.dist || manifest.revisionId
-      ? manifest.version
-      : `${Constants.nativeBuildVersion}`;
-
+  if (!nativeOptions.dist) {
+    // if the dist is already explicitly passed in, use that, otherwise:
```

Comment:
```suggestion
```

this is pretty much stated by the code as it is, so we can probably get rid of this comment